stix_shifter_modules/sentinelone/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,645 @@
+    {
+      "key": "x-oca-event.process_ref",
+      "object": "event",
+      "references": "process"
+    }
+  ],
+  "srcProcParentName": [
+    {
+      "key": "process.name",
+      "object": "parent_process"
+    },
+    {
+      "key": "process.parent_ref",
+      "object": "process",
+      "references": "parent_process"
+    }
+  ],
+  "tgtProcName": {
+    "key": "process.name",
+    "object": "process"
+  },
+  "srcProcName": {
+    "key": "process.name",
+    "object": "process"
+  },
+  "registryKeyPath": [
+  {
+  "key": "windows-registry-key.key",
+  "object": "registry_key"
+  },
+  {
+  "key": "x-oca-event.registry_ref",
+  "object": "event"
+  }
+  ],
+  "registryValue": {
+    "key": "windows-registry-key.values",
+    "object": "registry_key",
+    "transformer": "ToLowercaseArray"
+  },
+  "srcProcImagePath": {
+    "key": "process.extensions.x-sentinelone-process.image_path",
+    "object": "process"
+  },
+  "tgtProcImagePath": {
+    "key": "process.extensions.x-sentinelone-process.image_path",
+    "object": "process"
+  },
+  "srcProcStorylineId": {
+    "key": "process.extensions.x-sentinelone-process.story_line_id",
+    "object": "process"
+  },
+  "tgtProcStorylineId": {
+    "key": "process.extensions.x-sentinelone-process.story_line_id",
+    "object": "process"
+  },
+  "srcProcIntegrityLevel": {
+    "key": "process.extensions.x-sentinelone-process.integrity_level",
+    "object": "process"
+  },
+  "tgtProcIntegrityLevel": {
+    "key": "process.extensions.x-sentinelone-process.integrity_level",
+    "object": "process"
+  },
+  "srcProcUid": {
+    "key": "process.extensions.x-sentinelone-process.process_unique_id",
+    "object": "process"
+  },
+  "tgtProcUid": {
+    "key": "process.extensions.x-sentinelone-process.process_unique_id",
+    "object": "process"
+  },
+  "srcProcSignedStatus": {
+    "key": "process.extensions.x-sentinelone-process.signed_status",
+    "object": "process"
+  },
+  "tgtProcSignedStatus": {
+    "key": "process.extensions.x-sentinelone-process.signed_status",
+    "object": "process"
+  },
+  "srcProcPublisher": {
+    "key": "process.extensions.x-sentinelone-process.publisher",
+    "object": "process"
+  },
+  "tgtProcPublisher": {
+    "key": "process.extensions.x-sentinelone-process.publisher",
+    "object": "process"
+  },
+  "srcProcVerifiedStatus": {
+    "key": "process.extensions.x-sentinelone-process.verified_status",
+    "object": "process"
+  },
+  "tgtProcVerifiedStatus": {
+    "key": "process.extensions.x-sentinelone-process.verified_status",
+    "object": "process"
+  },
+  "srcProcReasonSignatureInvalid": {
+    "key": "process.extensions.x-sentinelone-process.signature_invalid_reason",
+    "object": "process"
+  },
+  "tgtProcReasonSignatureInvalid": {
+    "key": "process.extensions.x-sentinelone-process.signature_invalid_reason",
+    "object": "process"
+  },
+  "srcProcImageSha1": {
+    "key": "process.extensions.x-sentinelone-process.sha1_image",
```

Comment:
All of these image hashes should be file hashes. The process can reference the file through `binary_ref`

Reply:
It is  now mapped with file hashes.